src/main/java/com/teragrep/cfe_39/consumers/kafka/KafkaReader.java

```diff
@@ -45,31 +45,41 @@
  */
 package com.teragrep.cfe_39.consumers.kafka;
 
+import com.teragrep.cfe_39.configuration.ConfigurationImpl;
 import org.apache.kafka.clients.consumer.*;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.time.Duration;
+import java.time.Instant;
 import java.util.*;
 
 public class KafkaReader implements AutoCloseable {
```

Comment:
Could this be a final class?

Reply:
Solved in commit da226e27787f2ff3dbbb9ed7f7ad5d330b8fa4b5